eng/Versions.props

```diff
@@ -243,7 +242,7 @@
     <SystemIOFileSystemPrimitivesVersion>4.3.0</SystemIOFileSystemPrimitivesVersion>
     <SystemIOPipesAccessControlVersion>5.0.0</SystemIOPipesAccessControlVersion>
     <SystemIOPipelinesVersion>7.0.0</SystemIOPipelinesVersion>
-    <SystemManagementVersion>5.0.0-preview.8.20407.11</SystemManagementVersion>
+    <SystemManagementVersion>7.0.0</SystemManagementVersion>
```

Comment:
@jaredpar This one seems pretty safe. I think it's only used in RunTests.csproj

Reply:
Similar rationale as above so yes this is safe.